OJP/OJP_Trips.xsd

```diff
@@ -200,6 +181,181 @@
+			<xs:element name="SumCompleteTrip" type="xs:float" default="0" minOccurs="0">
+				<xs:annotation>
+					<xs:documentation>The preference has to be evaluated for each PathLink and the results are summed over the complete Trip. Convention: A value over 1000 means that an essential preference has been violated and the Trip is not valid.</xs:documentation>
+				</xs:annotation>
+			</xs:element>
+			<xs:element name="IncompleteDataButRealityMightBeBetter" type="xs:boolean" default="false" minOccurs="0">
+				<xs:annotation>
+					<xs:documentation>Indicates whether SumCompletTrip is based on incomplete data. Convention: In case of incomplete data, SumCompleteTrip is calculated by assuming an unfavorable evaluation.</xs:documentation>
+				</xs:annotation>
+			</xs:element>
+		</xs:sequence>
+	</xs:complexType>
+	<xs:simpleType name="PreferenceTypeEnumeration">
+		<xs:annotation>
+			<xs:documentation>Preference features the system is able to check or evaluate.</xs:documentation>
+		</xs:annotation>
+		<xs:restriction base="xs:string">
+			<xs:annotation>
+				<xs:documentation>Boolean features - they evaluate to 0 in the favorable case, to 1 in the unvaforable case.</xs:documentation>
+			</xs:annotation>
+			<xs:enumeration value="levelEntrance"/>
+			<xs:enumeration value="noSingleStep"/>
+			<xs:enumeration value="noStairs"/>
+			<xs:enumeration value="noRamp"/>
+			<xs:enumeration value="noRampUp"/>
+			<xs:enumeration value="noRampDown"/>
+			<xs:enumeration value="noElevator"/>
+			<xs:enumeration value="noEscalator"/>
+			<xs:enumeration value="noTravelator"/>
+			<xs:enumeration value="visualSigns"/>
+			<xs:enumeration value="tactileGuidingStrips"/>
+			<xs:enumeration value="tactileOrAuditorySigns"/>
+			<xs:enumeration value="theAdditionalConditions">
+				<xs:annotation>
+					<xs:documentation>The boolean conditions in Score.AdditionalConditions.</xs:documentation>
+				</xs:annotation>
+			</xs:enumeration>
+			<!-- The following seem less close to realisation:
+			<xs:enumeration value="onlyExtraReliableConnections"/>
+			<xs:enumeration value="onlyConnectionsWithBaggageTrolleys"/>
+			-->
+			<xs:annotation>
+				<xs:documentation>Numerical features - they evaluate to a value that will be multiplied by Preference.WeightFactor.</xs:documentation>
+			</xs:annotation>
+			<xs:enumeration value="numberOfChanges"/>
+			<xs:enumeration value="waitDuration"/>
+			<xs:enumeration value="walkDistance"/>
+			<xs:enumeration value="walkDistanceCovered"/>
+			<xs:enumeration value="walkDistanceNotCovered"/>
+			<xs:enumeration value="walkDistancePossiblyCovered"/>
+			<!-- The following seem less close to realisation:
+			<xs:enumeration value="distanceByCycle"/>
+			<xs:enumeration value="distanceByScooter"/>
+			<xs:enumeration value="price"/>
+			<xs:enumeration value="scenicRatingsMax"/>
+			<xs:enumeration value="scenicRatingsAvg"/>
+			<xs:enumeration value="environmentalFootprint"/>
+			-->
+		</xs:restriction>
+	</xs:simpleType>
```

Comment:
Example to illustrate a Score definition:

<Score>
	<ScoreType>sportyWheelchair</ScoreType>
	<Name>Wheelchair-sporty-typical</Name>
	<Intention>For wheelchair users managing single steps with relative ease; nevertheless try to avoid upward ramps.</Intention>
	<Priority>priority1</Priority>
	<Preference>
		<PreferenceType>noStairs</PreferenceType>
		<WeightFactor>1000</WeightFactor>
	</Preference>
	<Preference>
		<PreferenceType>noEscalator</PreferenceType>
		<WeightFactor>1000</WeightFactor>
	</Preference>
	<Preference>
		<PreferenceType>noSingleStep</PreferenceType>
		<WeightFactor>5</WeightFactor>
	</Preference>
	<Preference>
		<PreferenceType>noRampUp</PreferenceType>
		<WeightFactor>15</WeightFactor>
	</Preference>
	<Preference>
		<PreferenceType>noRampDown</PreferenceType>
		<WeightFactor>1</WeightFactor>
	</Preference>
	<Preference>
		<PreferenceType>walkDistance</PreferenceType>
		<WeightFactor>0.01</WeightFactor>
	</Preference>
	<Preference>
		<PreferenceType>levelEntrance</PreferenceType>
		<WeightFactor>5</WeightFactor>
	</Preference>
	<Preference>
		<PreferenceType>numberOfChanges</PreferenceType>
		<WeightFactor>10</WeightFactor>
	</Preference>
	<Preference>
		<PreferenceType>theAdditionalConditions</PreferenceType>
		<WeightFactor>1000</WeightFactor>
	</Preference>
	<AdditionalConditions>
		<Params>
			<PrivateModeFilterStructure>
				<Exclude>true</Exclude>
				<PrivateMode>carPooling</PrivateMode>
				<PrivateMode>cycle</PrivateMode>
				<PrivateMode>scooter</PrivateMode>
			</PrivateModeFilterStructure>
			<WalkSpeed>150</WalkSpeed>
			<AdditionalTransferTime>120s</AdditionalTransferTime>
		</Params>
	</AdditionalConditions>
</Score>